Gemfile

```diff
@@ -27,6 +27,7 @@ gem "simple_form"
 gem "slim"
 
 # all other gems
+gem "bootsnap"
```

Comment:
Do we need `bootsnap` in production?

Reply:
Regarding bootsnap README it's recommended to add to all envs.

https://github.com/Shopify/bootsnap#environments

Also, they say just add it to main gems group.